ROADMAP.md

```diff
@@ -0,0 +1,54 @@
+# developersIndia Website's Development Roadmap
+
+## Overview
+
+This document outlines our aspirational roadmap to deliver the development goals of the project. We anticipate substantial feedback from the community, and as such, this plan is subject to change.
+
+## Milestones
+
+The project is developed by the community & distributed on the Internet as described below.
+
+### Activities
+
+#### Dev Work
+
+* Fixes / Features for the developersIndia Website
+* Fixes / Features for future ideas bases on community feedback & needs
+
+#### Quality & Stability
+
+* Bug Fixes
+* Performance & Stability & Maintenance
+* Globalization, Localization, Internationalization, Accessibility
+* Tests
+
+#### Release
+
+* TODO: Insert domain over here when available
```

Comment:
do we want to add https://devsindia.in here? Or should be wait until the landing page is deployed 👀

Reply:
Can add right away to be honest but wording needs to be in such a way that people expects a "continuous work-in-progress project".